config/routes.rb

```diff
@@ -14,7 +14,42 @@
     end
   end
 
-  get '(:controller)/help', :action => 'welcome', :as => "help"
+  welcoming_controllers = [
```

Comment:
Do plugins need to extend this list?

Reply:
oh yes, https://github.com/search?q=org%3Atheforeman+welcome.html.erb&type=Code

Reply:
@mmoll I think this issue still stands, or at least the way I understand it users of a plugin with a welcome page will get 403 ? happy to be wrong..

Reply:
@ohadlevy correct, the affected plugins will need updates after this gets in.